src/plugins/sails.js

```diff
@@ -171,11 +171,12 @@ io.socket.on('game', function (evData) {
         case 'reLogin':
           store.dispatch('updateGameThenResetPNumIfNull', evData.data.game);
           break;
-        case 'requestStalemate':
-          if (evData.data.requestedByPNum !== store.state.game.myPNum && !evData.data.victory.gameOver) {
-            store.commit('setConsideringOpponentStalemateRequest', true);
-          }
+        case 'requestStalemate': {
+          const opponentConsideringStalemate = evData.data.requestedByPNum !== store.state.game.myPNum &&
+          !evData.data.victory.gameOver;
+          store.commit('setConsideringOpponentStalemateRequest', opponentConsideringStalemate);
```

Comment:
This closes the dialog for asking your opponent to stalemate when they respond to your request. It's redundant to the set functionality inside the OpponentRequestsStalemate dialog and might be better to just handle here.

It felt reasonable to overkill it just to make sure the damn thing closed but maybe I should remove the setter logic in the dialog for consistency. 

Notably I'm using the ternary assignment on 175 here to prevent the need for a conditional. That said it's now a multi-line boolean expression which may not be better. @seriouslysean penny for your thoughts

Reply:
I'm very in favor of doing this in one location. Remove it from the dialog and maybe leave a comment that it's handled here instead.